### PR TITLE
Adding na_ontap_disks.py Module

### PR DESCRIPTION
**Pull request by carchi8py**
##### SUMMARY
Adding na_ontap_disks.py Module

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME
- na_ontap_disks.py

##### ANSIBLE VERSION

```
bash-3.2# ansible --version
ansible 2.7.0.dev0 (sf_commonfiles caf71e24dd) last updated 2018/08/06 10:46:57 (GMT -700)
  config file = None
  configured module search path = [u'/Users/chrisarchibald/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /private/tmp/ansible/lib/ansible
  executable location = /private/tmp/ansible/bin/ansible
  python version = 2.7.12 (default, Oct 11 2016, 05:20:59) [GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.38)]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Review by schmots1 — APPROVED** [review]
shipit

File: lib/ansible/modules/storage/netapp/na_ontap_disks.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+  state:
+    description:
+    - Whether the specified aggregate should exist or not.
+    choices: ['present', 'absent']
+    default: 'present'
+
+  node:
+    required: true
+    description:
+    - Used only with auto or all parameter. It specifies the node
```
Comment (gundalow) [inline lib/ansible/modules/storage/netapp/na_ontap_disks.py]: This doesn't make sense to me (though I'm not a NetApp user)
* `Used only with auto or all parameter` - What defines `auto` or `all`?
* Is this meant to be the name of a node? does node mean disk, cluster, ...?

**Review by gundalow — CHANGES_REQUESTED** [review]

**Review by schmots1 — APPROVED** [review]
shipit

File: lib/ansible/modules/storage/netapp/na_ontap_disks.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+  node:
+    required: true
+    description:
+    - It specifies the node to assign all visible unowned disks.
+
+'''
+
+EXAMPLES = """
+- name: Assign unowned disks
```
Comment (gundalow) [inline lib/ansible/modules/storage/netapp/na_ontap_disks.py:38]: How would you unassign, set `node:` to empty, null? 
Would be worth adding an example for that. 

If it's not possible then I think `short_description` may need updating

**Review by gundalow — CHANGES_REQUESTED** [review]

**Comment by schmots1** [comment]
Updated.  Will change short description back for a later release when we modify how much this module does.

**Review by gundalow — APPROVED** [review]

**Pull request merged**